### PR TITLE
Add license file

### LINKED ISSUE #286: Please add a LICENSE file
Please add a LICENSE file -- The MIT license specifically states that the text of the license must accompany the source code.

### PR DESCRIPTION
**Pull request by jaredsmith**
Add a license file with the license text of the MIT license.

Fixes #286

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+### MIT License
+
+Copyright © [Steve Mao](https://github.com/stevemao)
```
Comment (stevemao) [inline LICENSE.md]: This should be the `conventional-changelog` team

**Comment by jaredsmith** [comment]
I've changed the copyright holder.  Is that acceptable?

**Comment by hbetts** [comment]
@jaredsmith I believe the License file must be included within the sub-directory of each individual package so that when `npm publish` is executed within each sub-package, the License file will included in the published artifact.

**Comment by stevemao** [comment]
Can you rebase and squash the commits?

**Comment by jaredsmith** [comment]
OK, I've rebased and squashed the commits.

File: packages/conventional-changelog-angular/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+### MIT License
```
Comment (hbetts) [inline packages/conventional-changelog-angular/LICENSE.md]: The `conventional-changelog-angular` package uses the ISC license.

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+### MIT License
```
Comment (hbetts) [inline LICENSE.md]: The top-level `package.json` file specifies an ISC license.

File: packages/conventional-changelog-atom/LICENSE.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+### MIT License
```
Comment (hbetts) [inline packages/conventional-changelog-atom/LICENSE.md]: The `conventional-changelog-atom` package specifies an ISC license.

**Comment by jaredsmith** [comment]
Good call -- I assumed they were all under the same license, but that was a bad assumption.  The PR should reflect the actual license of each package now.

**Comment by hbetts** [comment]
> I assumed they were all under the same license

Our current license situation is definitely confusing. It's my hope that we'll consolidate to a single license across all Conventional Changelog projects. However that will likely require legal review.

This change, however, is a necessary first step in improving our licensing compliance. So thank you for putting in the work on this pull request.

:+1: This change looks good to me.

**Comment by hbetts** [comment]
Successfully published:
 - conventional-changelog-angular@3.0.4
 - conventional-changelog-atom@0.2.7
 - conventional-changelog-cli@1.3.19
 - conventional-changelog-codemirror@0.3.7
 - conventional-changelog-core@2.0.8
 - conventional-changelog-ember@0.3.9
 - conventional-changelog-eslint@1.0.8
 - conventional-changelog-express@0.3.6
 - conventional-changelog-jquery@1.3.7
 - conventional-changelog-jshint@0.3.7
 - conventional-changelog-preset-loader@1.1.8
 - conventional-changelog-writer@3.0.7
 - conventional-changelog@1.1.21
 - conventional-commits-parser@2.1.7
 - conventional-recommended-bump@2.0.8
 - git-raw-commits@1.3.6
 - git-semver-tags@1.3.6
 - gulp-conventional-changelog@1.1.21
 - standard-changelog@1.0.16

**Pull request merged**